rules/aws_cloudtrail_rules/signal_-_aws_console_sign-in.yml

```diff
@@ -0,0 +1,24 @@
+AnalysisType: rule
+Filename: signal_-_aws_console_sign-in.py
+RuleID: "SIGNAL.-.AWS.Console.Sign-In"
```

Comment:
We already have an `AWS.Console.Login rule`, but it looks like this implementation triggers specifically for SSO-initiated logins. if that's the case, we should rename this rule to something like `AWS.Console.Login.SSO`